_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Announcing The Ombu Labs Shop!"
+date: 2016-02-04 18:35:00
+categories: ["Design"]
+author: "cecilia"
+---
+
+We recently worked on a series of Christmas presents for the team. We are happy to share part of the process and its final result!
+
+<!--more-->
+
+### Our process: it all started with an idea
+
+As it happens with every product, it starts with an idea. We wanted to make a t-shirt, a notebook and a tote bag that could reflect our passion for work and design.
+
+For the notebook, we chose a quote by Otl Aicher’s: “Good Art Inspires, Good Design Motivates”. Here you can see part of the design process, with some tweaks and final adjustments.
+
+<img src="/blog/assets/images/ombu-labs-shop/notebook-design-process.png" alt="Notebook Design Process" class="full-img">
+
+For our t-shirt we chose a very well known quote that inspires us day by day at work: Done is Better than Perfect. In this case, we started with some paper sketches and then its design:
+
+<img src="/blog/assets/images/ombu-labs-shop/lettering-1.jpg" alt="T-Shirt Design Process" class="full-img">
+<img src="/blog/assets/images/ombu-labs-shop/lettering-2.png" alt="Done is Better than Perfect" class="full-img">
+
+
+### Our Products
+After all this creative process, these are the results:
+
+<img src="/blog/assets/images/ombu-labs-shop/notebook.png" alt="Ombu Labs Notebook" class="full-img">
+
+<img src="/blog/assets/images/ombu-labs-shop/t-shirt.png" alt="Ombu Labs t-shirt" class="full-img">
+
+And last, but not least, our tote bag, which design was inspired by our day-to-day work and methodology. We made some illustrations and then created a final composition.
```

Comment:
@cecilia-ombulabs instead of "which design was" it should be "its design was inspired"